lib/accessLicenceClient.js

```diff
@@ -2,69 +2,76 @@
 
 /**
  * Access Licence Service Client
- *
  */
 const fetch = require('fetch-retry-or-die');
 const log = require('@financial-times/n-logger').default;
-const config = require('./config');
-const statusErrors = require('./statusErrors');
+const config = require('./helpers/config');
 const qs = require('querystring');
+const helpers = require('./helpers/helpers');
 
-const options = Object.assign({}, config.fetchOptions, {headers: Object.assign({}, config.fetchOptions.headers, {"X-API-KEY": config.accLicenceKey})});
-
-module.exports = {
-    getLicences,
-	getIDs,
-	getSeats
-};
+const options = Object.assign({}, config.fetchOptions);
```

Comment:
You could use the spread operator `...` here instead of `Object.assign()`:

`const options = {...config.fetchOptions};`